src/dawn/engine_dawn.cc

```diff
@@ -151,11 +188,19 @@ void HandleBufferMapCallback(DawnBufferMapAsyncStatus status,
   }
 }
 
+uint32_t Align(uint32_t value, size_t alignment) {
```

Comment:
Needs a method comment.

// Returns |value| but rounded up to a multiple of |alignment|.  |alignment| is assumed to be a power of 2.